common/src/main/cpp/extensions/openxr_fb_passthrough_extension_wrapper.cpp

```diff
@@ -607,6 +639,137 @@ bool OpenXRFbPassthroughExtensionWrapper::has_layer_depth_passthrough_capability
 	return (system_passthrough_properties.capabilities & XR_PASSTHROUGH_CAPABILITY_BIT_FB) && (system_passthrough_properties.capabilities & XR_PASSTHROUGH_CAPABILITY_LAYER_DEPTH_BIT_FB);
 }
 
+void OpenXRFbPassthroughExtensionWrapper::set_color_lut(float p_weight, const Ref<OpenXRMetaPassthroughColorLut> &p_color_lut) {
+	if (!meta_passthrough_color_lut_ext) {
+		UtilityFunctions::print("Passthrough color LUT extension not enabled!");
+		return;
+	}
+
+	if (p_color_lut->get_handle() == XR_NULL_HANDLE) {
+		create_color_lut(p_color_lut);
+	}
+
+	color_lut_handle = p_color_lut->get_handle();
+
+	current_passthrough_filter = PASSTHROUGH_FILTER_COLOR_MAP_LUT;
+	color_map_lut.colorLut = color_lut_handle;
+	color_map_lut.weight = CLAMP(p_weight, 0.0, 1.0);
+	passthrough_style.next = &color_map_lut;
+
+	if (is_passthrough_started()) {
+		XrResult result = xrPassthroughLayerSetStyleFB(passthrough_layer[current_passthrough_layer], &passthrough_style);
+		if (XR_FAILED(result)) {
+			UtilityFunctions::print("Failed to set passthrough style, error code: ", result);
```

Comment:
nit: let's start making better use of `printerr`. I've taken a quick look at the codebase and we don't seem to use it at all, so not a blocker for this PR, but let's start making use of it in scenarios where it makes sense.

Reply:
In normal engine code, I feel like we'd use `ERR_PRINT()` or `ERR_FAIL_COND()` in this sort of situations. However, at the moment, 'godot_openxr_vendors' has lots of `UtilityFunctions::print()` similar to this (and, actually, even the 'openxr' module in Godot has more `print_line()` for errors than it probably should - I think this pattern probably followed over from there).

Reply:
yeah, I've noticed that in the code I write myself for this plugin, I tend to default to `print` no matter the circumstances, but I also think it's because I know how to interpret a message, whether it'd be a verbose output or an error message, when I see one.

This may not be so obvious to end users though, which is where distinguishing the output comes handy. e.g: end users can use the editor's output window to isolate (and report) errors.

Reply:
Updated to `printerr` in all the places in this commit where I believe it makes sense!